widget/embedded/src/QueueManager.tsx

```diff
@@ -29,7 +29,7 @@ function QueueManager(props: PropsWithChildren) {
 
   const swapQueueDef = useMemo(() => {
     return makeQueueDefinition({
-      API_KEY: getConfig('API_KEY'),
+      API_KEY: props.apiKey || getConfig('API_KEY'),
```

Comment:
Shouldn't we add `props.apiKey` to `useMemo` dependencies.

Reply:
fixed.